### PR TITLE
Meappy

### PR DESCRIPTION
**Pull request by walt-r**
This adds a new MEAPPY directory and an importable module for loading MED64 data.

### CODE REVIEW

File: meappy/med64_data.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+lag_bins_ms = np.arange(-lag_bins, lag_bins + 1) * (1000 / bins_per_sec)
+
+
+
+def find_expt_files(data_dir):
+    files = os.listdir(data_dir)
+
+    re_units = re.compile(r'^units.*')
+    re_tx = re.compile(r'^treatmentinfo.*')
+    re_expr_id = re.compile(r"([0-9]{3}_[0-9]{2}h[0-9]{2}m[0-9]{2}s)\.['csv|mat']{3}")
```
Comment (MadelineMartin) [inline meappy/med64_data.py:24]: what does this do?

Comment (walt-r) [reply meappy/med64_data.py:24]: Those are RegEx expressions. That allow matching to certain text patterns. These are to identify the different files in the med64 data folder in an automated way.  Perhaps not needed now that we have the YAML file. I may remove them.

**Comment by MadelineMartin** [comment]
looks good!

**Comment by walt-r** [comment]
Here is a squash and merge. Lets see how to re-write the commit message.

**Pull request merged**